keyboards/trnthsn/e8ghty/info.json

```
@@ -0,0 +1,585 @@
+{
+    "manufacturer": "ThanhSon.Mech",
+    "keyboard_name": "E8ghty",
+    "maintainer": "trnthsn",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "matrix_pins": {
+        "cols": ["A8", "B14", "B12", "B11", "B10", "B1", "B0", "A7", "A6", "A5", "A3", "A2", "A1", "A0", "C14", "C13", "B5"],
+        "rows": ["A15", "A10", "A9", "A4", "C15", "B9"]
+    },
+    "rgblight": {
+        "animations": {
+            "alternating": true,
+            "breathing": true,
+            "christmas": true,
+            "knight": true,
+            "rainbow_mood": true,
+            "rainbow_swirl": true,
+            "snake": true,
+            "static_gradient": true,
+            "twinkle": true
+        },
+        "led_count": 26,
+        "sleep": true
+    },
+    "ws2812": {
+        "pin": "B15",
+        "driver": "spi"
+    },
+    "indicators": {
+        "caps_lock": "B4"
+    },
+    "url": "",
+    "usb": {
+        "vid": "0x5453",
+        "pid": "0x3830",
+        "device_version": "0.0.1"
+    },
+    "layout_aliases": {
+        "LAYOUT": "LAYOUT_all"
+    },
+    "community_layouts": [
+        "tkl_f13_ansi_tsangan_split_bs_rshift",
+        "tkl_f13_ansi_tsangan",
+        "tkl_ansi_tsangan",
+        "tkl_ansi_tsangan_split_bs_rshift"
+    ],
```

1. remove `layout_aliases`; as it's not used in keymap(s)
2. tidy community layouts literal array
```suggestion
    "community_layouts": ["tkl_ansi_tsangan", "tkl_ansi_tsangan_split_bs_rshift", "tkl_f13_ansi_tsangan", "tkl_f13_ansi_tsangan_split_bs_rshift"],
```